src/Model/Behavior/GeocoderBehavior.php

```diff
@@ -37,22 +38,25 @@
  */
 class GeocoderBehavior extends Behavior {
 
+	public const OPTION_COORDINATES = 'coordinates';
+	public const OPTION_LAT = 'lat';
+	public const OPTION_LNG = 'lng';
+
 	/**
 	 * @var array
 	 */
 	protected $_defaultConfig = [
 		'address' => null,
 		'allowEmpty' => true,
 		'expect' => [],
-		'lat' => 'lat', // Field name in your entity
-		'lng' => 'lng', // Field name in your entity
+		'lat' => self::OPTION_LAT, // Field name in your entity
+		'lng' => self::OPTION_LNG, // Field name in your entity
 		'formattedAddress' => 'formatted_address', // Field name in your entity
 		'addressFormat' => '%S %n, %z %L', // For class StringFormatter
 		'locale' => null, // For GoogleMaps provider
 		'region' => null, // For GoogleMaps provider
 		'ssl' => true, // For GoogleMaps provider
-		//'bounds' => '',
-		'overwrite' => false, // Overwrite existing
+		'overwrite' => true, // Overwrite existing
```

Comment:
This is just preparation for the actual implementation later right?

Reply:
I basically just fixed the docs and with it the line here to match expectations.